.gitignore

```diff
@@ -1,8 +1,3 @@
 .idea
-cache.properties
-config.php
 composer.lock
 vendor
-.DS_Store
```

Comment:
Ty vyhodis ignorovani metadat, ktery si uklada OSX a KDevelop IDE. Ale pro PHPStorm to tam nechas ?

Reply:
udelal jsem to z gitignore z templaty https://github.com/keboola/component-generator/blob/master/templates/php-component/.gitignore

kdevelop jeste nekdo pouziva? :)

Reply:
Pokud ty soubory negeneruje přímo aplikace, ale tvůj lokální systém, tak je lepší to dát do ~/.gitignore globálního ;) 

Třeba Symfony v default nemá ani .idea složku v ignore https://github.com/symfony/symfony/blob/master/.gitignore

Viděl jsem i projekty, kde se .idea verzovalo. A náhodou to není úplně špatný, když někdo nastaví pro ten projekt třeba spouštění testů přímo ze Stormy ;)

Reply:
Pak teda by tam nemelo byt ani .idea :-)

Reply:
klidne, ale tak to pojdme zmenit v https://github.com/keboola/component-generator/tree/master/templates/php-component anebo #best_practices